editors/sc-ide/core/doc_manager.cpp

```diff
@@ -773,7 +773,7 @@ void DocumentManager::handleNewDocScRequest( const QString & data )
             Q_EMIT( opened(document, 0, 0) );
         }
     } catch (std::exception const& e) {
-        qWarning() << "DocumentManager::" << __func__ << ": could not handle request:" << e.what();
+        qWarning() << "DocumentManager::" << __FUNCTION__ << ": could not handle request:" << e.what();
```

Comment:
it's supported by gcc, clang, and msvc. despite `__function__` being 'standard', it's not supported by msvc